lib/ui/text.dart

```diff
@@ -3014,6 +3014,28 @@ abstract class ParagraphBuilder {
   /// [Paragraph].
   factory ParagraphBuilder(ParagraphStyle style) = _NativeParagraphBuilder;
 
+  /// Whether the rounding hack enabled by default in SkParagraph and TextPainter
+  /// is disabled.
+  ///
+  /// Do not rely on this getter as it exists for migration purposes only and
+  /// will soon be removed.
+  static bool get shouldDisableRoundingHack {
```

Comment:
Can we mark this one as `@deprecated` to give a stronger signal that it is not to be used? That's how we usually mark temporary things for migration purposes in the framework.